rust_src/build.rs

```diff
@@ -497,7 +497,7 @@ fn build_ignored_crates(path: &PathBuf) -> bool {
 fn generate_crate_exports(path: &PathBuf) -> Result<(), BuildError> {
     let modules = find_crate_modules(&path.join("src"))?;
 
-    fs::create_dir(path.join("out"));
+    let _ = fs::create_dir(path.join("out"));
```

Comment:
Do you think it makes sense to return `BuildError` when `crate_dir` fails ?

Reply:
It makes sense.
I try it. If the directory exists, this creation return error too.  That breaks our build process.